### ISSUE TITLE
Scope of the glossary

### ISSUE THREAD

**Issue by Beanow** [sourcecred/docs#5]
The idea of moving the glossary to this repository was mentioned.
Though there is one aspect I'd like some thoughts on.

Especially from @vsoch and @Lex-Birdie.

I think there's a spectrum of terms, some are adopted as-is and are also used elsewhere. There may be good references for it online. For example "[bikeshedding](https://css-tricks.com/what-is-bikeshedding/)".

Others do not make sense at all outside the context of our community. Such as "[Grain](https://discourse.sourcecred.io/t/an-overview-of-grain/147)".

And there is some middle ground, where we might have modified it in our context, but is based on a general idea that's used elsewhere. Like a "[Champion](https://discourse.sourcecred.io/t/about-champions-and-heroes/291)".

In other words, the spectrum is about:
The way we use the term, _how specific is it to our community context?_

Should we make distinctions in the glossary for this?
Should we track the ones that are not specific to us?

Interested in hearing your thoughts on this.

**Comment by vsoch** [comment]
Yeah this is a great idea! I think logically a glossary of terms fits very logically with docs. I can see myself writing documentation, using a word like bikeshedding, realizing it might not be known, and then adding it to the glossary of terms in the docs (for the same PR) with a link. It would also be very easy for a reviewer to look at some update, see a term he/she doesn't recognize, and then request it be added and linked to the glossary for the PR. If we don't do it like that, then we have to maintain the terms and docs using them in two separate places, which imho makes it harder / more confusing to contribute.

In terms of context, I think it's reasonable to say "This is what you'd normally thing of a champion, in sourcecred it means this."

**Comment by decentralion** [comment]
I feel like there might be two docs here: a "User Glossary" and a "Contributor Glossary".

The "User Glossary" is intended for people who want to understand SourceCred from the perspective of using it in their projects, etc. So we'd want entries for "cred", "grain", "champion", etc.

The "Contributor Glossary" (or "Community Glossary"?) is for people who want to participate actively as part of the SC community itself. So terms like bikeshedding would be most relevant there.

**Comment by vsoch** [comment]
This just came up for me working locally - I was about to add a "terms" section to the bottom of a file, and then remembered this issue. Maybe this would make sense to work on before (what I'm currently working on) for easier reference, I'll take a look at this glossary idea in more detail this week.
